chirp/drivers/radtel_t18.py

```diff
@@ -1020,7 +1034,7 @@ class RT22SRadio(T18Radio):
                     chirp_common.PowerLevel("Low",  watts=0.50)]
 
     _magic = b"9COGRAM"
-    _fingerprint = b"SMP558" + b"\x02"
+    _fingerprint = [b"SMP558" + b"\x02", ]
```

Comment:
Convention would say that you should make a list of one be `[thing]` not `[thing, ]`. Some stricter pep8 checkers might fail you on this :)

Reply:
OK. I just just seen quite a few lists done this way so I thought it was some convention or something. I will fix it.

Reply:
Well, we might have more in the chirp code, but... in the grand scheme of cosmic python convention, it's unusual. No need to fix it necessarily, I'm just saying...

Reply:
Oh.. I didn't notice that you had already merge it.